pages/views/sitemap.njk

```diff
@@ -0,0 +1,19 @@
+---
+permalink: "/sitemap.xml"
+eleventyExcludeFromCollections: true
+---
+<?xml version="1.0" encoding="UTF-8"?>
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
+  {%- for page in collections.all %}
+  {%- if page.data.ignore != true %}
```

Comment:
if we only want to exclude drafts collection, mb we can use this line `{%- if not (page.data.tags == 'drafts') %}` (and we don't need to add to drafts `ignore: true`)

Reply:
It's really perfect syntax. Thanks